src/shared/components/setup-locker-demo/SetupLocker.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```

Comment:
Can you delete all the demo files?

Reply:
Yes, I forgot to do it. At this point we can get rid of the demo files we needed to have earlier for testing purposes